From b54281b899ac1932bc6124b8e57433fccc0b4c80 Mon Sep 17 00:00:00 2001
From: Daniel Baluta <daniel.baluta@nxp.com>
Date: Thu, 8 Mar 2018 19:04:19 +0200
Subject: [PATCH 3461/5242] MLK-17734-2: ASoC: fsl: ak5558: Remove support for
 192KHz in TDM mode

commit  df1e16ea1998eca67c05dd0a77e114b256cf0b57 from
https://source.codeaurora.org/external/imx/linux-imx.git

Using TDM256 mode (our only supported mode) in order to
support 192KHz we would need a MCLK of 192000 * 512 = 98304000.

But maximum frequency supported by the Audio PLL is 4.91 MHz.

Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/imx-ak5558.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/fsl/imx-ak5558.c b/sound/soc/fsl/imx-ak5558.c
index 70075db..84cd2e9 100644
--- a/sound/soc/fsl/imx-ak5558.c
+++ b/sound/soc/fsl/imx-ak5558.c
@@ -80,7 +80,7 @@ struct imx_ak5558_fs_mul {
 
 static const u32 ak5558_tdm_rates[] = {
 	8000, 16000, 32000,
-	48000, 96000, 192000,
+	48000, 96000
 };
 
 static const u32 ak5558_channels[] = {
-- 
1.7.9.5

